scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/nnframes/TreeModelUtils.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.dllib.nnframes
+import com.intel.analytics.bigdl.dllib.utils.Log4Error
+import com.microsoft.azure.synapse.ml.lightgbm.{LightGBMClassifier => MLightGBMClassifier}
+import com.microsoft.azure.synapse.ml.lightgbm.{LightGBMRegressor => MLightGBMRegressor}
+
+
+object TreeModelUtils {
+
+  def convert2CamelCase(pythonStyle: String): String = {
+    val data = pythonStyle.split("_")
+    data(0) + data.slice(1, data.size)
+      .map(x => x.substring(0, 1).toUpperCase() + x.substring(1)).mkString("")
+  }
+
+  def setParams(lgbmEstimator: Any, lgbmParams: Map[String, Any]): Unit = {
```

Comment:
maybe we can limit lgbmEstimator to LGBMModel type?

Reply:
> maybe we can limit lgbmEstimator to LGBMModel type?

there is a LightGBMBase, but it is Trait

Reply:
throwing error message if not supported in  https://github.com/intel-analytics/BigDL/pull/5789